adapter/pkg/api/restserver/config_restapi.go

```diff
@@ -75,14 +75,15 @@ func configureAPI(api *operations.RestapiAPI) http.Handler {
 		return &p, nil
 	}
 
-	api.APIIndividualPostImportAPIHandler = api_individual.PostImportAPIHandlerFunc(func(params api_individual.PostImportAPIParams, principal *models.Principal) middleware.Responder {
+	api.APIIndividualPostApisHandler = api_individual.PostApisHandlerFunc(func(params api_individual.PostApisParams,
```

Comment:
Add the readme on how to generate this auto generated codes

Reply:
added via https://github.com/wso2/product-microgateway/pull/1525/commits/f1442ebc609d8cae72391d928268f8ae1e9fd183